### DISCUSSION TITLE
Unable to build on Rocky Linux 8

### DISCUSSION THREAD

**Discussion by rith88** [AcademySoftwareFoundation/OpenRV#104] [Build Issues]
Hi,
Anybody know how to build this on Rocky Linux.
I am getting this error

License: LGPL version 2.1 or later
config.h is unchanged
config.asm is unchanged
libavutil/avconfig.h is unchanged
libavfilter/filter_list.c is unchanged
libavcodec/codec_list.c is unchanged
libavcodec/parser_list.c is unchanged
libavcodec/bsf_list.c is unchanged
libavformat/demuxer_list.c is unchanged
libavformat/muxer_list.c is unchanged
libavdevice/indev_list.c is unchanged
libavdevice/outdev_list.c is unchanged
libavformat/protocol_list.c is unchanged
ffbuild/config.sh is unchanged
ninja: build stopped: subcommand failed.

Thanks.

**Comment by savoiei** [comment]
Hi rith88,

It is in the plans to support building on Rocky Linux 8 eventually, but as of now, only CentOS 7 as been certified as a Linux distribution to build OpenRV.

Thanks,
Ian

**Comment by rogernelson** [comment]
There's no actual error in your output, but I notice it's failing to build ffmpeg. I also failed here building on Rocky because ffmpeg's build scripts are using tcsh and it's not installed on Rocky by default.  You can try `sudo dnf install tcsh` and hopefully that will fix your problem.

I'm currently writing a build guide for Rocky 9, which I hope to finish soon.  hopefully that will help you out if you're still stuck.

**Reply by fathomrhys** [reply]
Hi Roger, 
Very keen to read your Rocky9 guide when you are done. Currently stuck trying to build for it.

**Reply by geffrak** [reply]
@fathomrhys, the guide is here: https://github.com/AcademySoftwareFoundation/OpenRV/blob/main/docs/build_system/config_linux_rocky9.md

**Reply by rogernelson** [reply]
Just a note that I wrote the guide for Rocky 9 and not 8. But I think it should  be mostly similar. If you get blocked let me know and I can put one together for Rocky 8 as well.

**Comment by fathomrhys** [comment]
Hi Roger,

Thanks very much for this!

It’s Rocky 9 that I want to build for anyway.

On Sat, 13 May 2023 at 11:35 AM, Roger Nelson ***@***.***>
wrote:

> Just a note that I wrote the guide for Rocky 9 and not 8. But I think it
> should be mostly similar. If you get blocked let me know and I can put one
> together for Rocky 8 as well.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AcademySoftwareFoundation/OpenRV/discussions/104#discussioncomment-5889403>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2SNWY5VOTIELOE4QHGVUNDXF3CKZANCNFSM6AAAAAAWIXWCUU>
> .
> You are receiving this because you were mentioned.Message ID:
> <AcademySoftwareFoundation/OpenRV/repo-discussions/104/comments/5889403@
> github.com>
>
-- 
  *Rhys Dippie*

*  VFX Supervisor*
*   www.fathomvfx.com <http://www.fathomvfx.com>*

*   mob: +64 27 2311232*
